deploy/base/gradle-v0.1.yaml

```diff
@@ -0,0 +1,362 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
so fyi, assuming my e2e change merges before this, I would expect your PR here to add an e2e similar to the maven based one I initially introduced where, but it leverages this gradle file and the associated run-gradle.yaml example you have.